alembic/env.py

```diff
@@ -0,0 +1,84 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from City_CRUD_API.models import *
+from Temperature_API.models import *
```

Comment:
Similar to the previous comment, avoid using wildcard imports for the Temperature API models. Import only the required models explicitly to maintain clarity.